components/library/gamin/Makefile

```diff
@@ -44,12 +41,17 @@ COMPONENT_PREP_ACTION= ( cd $(@D)  && \
 	automake -c -f -a &&\
 	autoconf )
 
+PYV += `echo $(PYTHON_VERSION) | tr -d .`
+COMPONENT_POST_INSTALL_ACTION.64 += ( cd $(PROTO_DIR)/$(PYTHON_VENDOR_PACKAGES) && \
+					$(MV) _gamin.so _gamin.cpython-$(PYV).so )
+COMPONENT_POST_INSTALL_ACTION += $(COMPONENT_POST_INSTALL_ACTION$(BITS))
+
 CONFIGURE_OPTIONS += --sysconfdir=/etc
 CONFIGURE_OPTIONS += --libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
 CONFIGURE_OPTIONS += --disable-static
-CONFIGURE_OPTIONS += --with-python=$(PYTHON.$(BITS))
-
-CONFIGURE_ENV += pythondir=$(PYTHON_VENDOR_PACKAGES)
+CONFIGURE_OPTIONS.32 += --with-python=no
+CONFIGURE_OPTIONS.64 += --with-python=/usr
+CONFIGURE_OPTIONS += $(CONFIGURE_OPTIONS$(BITS))
```

Comment:
Here you are likely missing a dot.  OTOH, I think this line is not needed because it is already done `make-rules/configure.mk`.